pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -222,6 +209,19 @@ func TestOpenShiftClusterStaticValidateClusterProfile(t *testing.T) {
 			},
 			wantErr: "400: InvalidParameter: properties.clusterProfile.resourceGroupId: The provided resource group '/subscriptions/7a3036d1-60a1-4605-8a41-44955e050804/resourcegroups/test-cluster' is invalid: must be in same subscription as cluster.",
 		},
+		{
```

Comment:
should be in a separate `TestOpenShiftClusterStaticValidateConsoleProfile` function

Reply:
done